### PR TITLE
Added a new Swag Entry (JetBrains)

### PR DESCRIPTION
**Pull request by Archis19**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -335,11 +335,11 @@
...
-    "dateAdded": "2020-09-30T03:19:14.000Z",
-    "tags": ["clothing", "hacktoberfest", "stickers"]
+    "description": "Get swag for contributing in general. At the end of the month-long event, the individual who makes the most significant contributions will receive a brand new laptop.",
+    "reference": "https://blog.mayadata.io/mayadata-celebrates-hacktoberfest-2019",
+    "image": "https://blog.mayadata.io/hs-fs/hubfs/Hacktoberfest-11-1.png",
+    "dateAdded": "2019-10-02T16:48:14.000Z",
+    "tags": ["clothing", "device", "hacktoberfest", "stickers", "expired"]
   },
   {
     "name": "Microsoft",
```
Comment (vikaspotluri123) [inline data.json:345]: Please revert these changes 😃

Comment (Archis19) [reply data.json:345]: sorry for the inconvenience, I have made the appropriate changes and created a new PR
